src/data/markdown/translated-guides/en/07 Testing Guides/05 Running distributed tests.md

```diff
@@ -0,0 +1,379 @@
+
+```yaml
+apiVersion: k6.io/v1alpha1
+kind: K6
+metadata:
+  name: run-k6-with-vars
+spec:
+  parallelism: 4
+  script:
+    configMap:
+      name: my-test
+      file: test.js
+  runner:
+    env:
+      - name: MY_CUSTOM_VARIABLE
+        value: 'this is my variable value'
+    envFrom:
+     - configMapRef:
+         name: my-config-vars
+     - secretRef:
+         name: my-secrets-vars
+```
+
+</CodeGroup>
+
+<Blockquote mod="note" title="">
+
+The above YAML introduces the `runner` section. This section applies to each pod that will be running a portion of your test, based upon the desired `parallelism`.
+
+</Blockquote>
+
+Now, with the referenced resources, our test scripts can [use environment variables](/using-k6/environment-variables/) as in the following:
+
+```javascript
+export function setup() {
+  console.log(`Variable is set as: ${__ENV.MY_CUSTOM_VARIABLE}`);
+}
+```
+
+### Change command-line arguments
+
+[k6 options](/using-k6/k6-options/) can be specified in many ways, one being the command-line.
+Specifying options via command-line can still be accomplished when using the operator as shown with the following example:
+
+<CodeGroup labels={["run-k6-with-args.yaml"]} lineNumbers={[]} showCopyButton={[true]}>
+
+```yaml
+apiVersion: k6.io/v1alpha1
+kind: K6
+metadata:
+  name: run-k6-with-args
+spec:
+  parallelism: 4
+  script:
+    configMap:
+      name: my-test
+      file: test.js
+  arguments:  --tag testid=run-k6-with-args --log-format json
+```
+
+</CodeGroup>
+
+With the above arguments, we're adding a [test-wide custom tag](/using-k6/tags-and-groups/#test-wide-tags) to metrics and changing the output format of logs to [JSON](/misc/glossary/#json).
+
+<Blockquote mod="note" title="">
+
+Be sure to visit the [options reference](https://k6.io/docs/using-k6/k6-options/reference/) for a listing of available options.
+
+</Blockquote>
+
+## 5. Run your test
+
+Tests are executed by applying the custom resource `K6` to a cluster where the operator is running.
+The test configuration is applied as in the following:
+
+```shell
+kubectl apply -f /path/to/your/k6-resource.yaml
+```
+
+After completing a test run, you need to clean up the test jobs created. 
+This is done by running the following command:
+
+```shell
+kubectl delete -f /path/to/your/k6-resource.yaml
+```
+
+<Blockquote mod="note" title="Best Practice">
+
+If you make use of [real-time results output](/results-output/real-time/), e.g. [Prometheus Remote Write](/results-output/real-time/prometheus-remote-write/), use the `cleanup` option to automatically remove resources upon test completion as with the following example:
+
+```yaml
+apiVersion: k6.io/v1alpha1
+kind: K6
+metadata:
+  name: run-k6-with-realtime
+spec:
+  parallelism: 4
+  # Removes all resources upon completion
+  cleanup: post
+  script:
+    configMap:
+      name: my-test
+      file: test.js
+  arguments:  -o experimental-prometheus-rw
+```
```

Comment:
This example is great, thanks! Btw, it's the first time we have it documented: its absent even in Readme ATM :smile: